clap.pc.in

```diff
@@ -0,0 +1,8 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
+
+Name: clap
+Description: The interface headers for the CLAP audio plugin API
```

Comment:
Bikeshedding is welcome here. The description field is mandatory, so I had to put in something.